packages/bolt-connection/test/connection-provider/connection-provider-direct.test.js

```diff
@@ -222,10 +222,10 @@ it('should change error to retriable when error when TokenExpired happens and st
 
   const error = conn.handleAndTransformError(expectedError, address)
 
-  expect(error.retriable).toBe(true)
+  expect(error.retryable).toBe(true)
```

Comment:
Up to you: might be worth testing both (deprecated and renamed) attributes. (more places this applies to, won't comment again)